include/_kernel.h

```diff
@@ -55,6 +55,10 @@
 /* must come after _enable_all_exts.h b/c of pocl_types.h*/
 #include "_kernel_c.h"
 
+#define __kernel_exec(X, typen) \
```

Comment:
Please add a comment what is the purpose of this macro with two leading underscores.